extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -69,6 +70,12 @@
     @ConfigItem(defaultValue = "10s")
     public Duration connectionTimeout = Duration.ofSeconds(10);
 
+    /**
+     * The maximum size of the connection pool used by the WebClient
+     */
+    @ConfigItem
+    public OptionalInt webClientMaxPoolSize = OptionalInt.empty();
```

Comment:
Isn't `connectionTimeout` also related to the web client? If so, being consistent and dropping the `webClient` prefix would make sense.

Let's decide this before merging.

Reply:
Yes this is related, it is used in the same method to set `WebClientOption`

Reply:
@gsmet, @glefloch sure, `maxPoolSize` works as well, thanks

Reply:
I updated the property name, and all related reference (getter/setter, ...)